src/dxc_helper.cc

```diff
@@ -0,0 +1,146 @@
+// Copyright 2019 The Amber Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "src/dxc_helper.h"
+
+#include <algorithm>
+#include <sstream>
+
+#include "third_party/dxc/include/dxc/Support/Global.h"
```

Comment:
This seems over-descriptive.  In future we should allow DXC's source to be in some configurable location.  But that's for the future.

Reply:
Include paths should be from the root src/ directory. Or are you thinking we should allow DXC to live outside amber?

Reply:
I mean DXC should be able to live outside of Amber.  Like, if I had one giant build tree that included many tools for the Vulkan shader stack.
So, I'd have 

#include "dxc/Support/Global.h" 

so that I can always find with a -I option on the compiler, where that is controlled from a CMake string expression.